packages/ui/__stories__/Link.stories.tsx

```diff
@@ -100,3 +100,29 @@ export const SmallWithIconActive = () => (
     Small Text Link
   </Link>
 )
+
+// A few 'secondary' kind variations:
+
+export const SecondaryNormal = () => (
```

Comment:
Could you generate screenshots for these stories? `npm run generate-screenshots`

Reply:
@aigoncharov how about now? (sorry bit of a foreign workflow for me, will take some time to adapt ;))